src/commands/init.js

```diff
@@ -30,9 +31,9 @@ const templates = [{
  *
  * @returns {Promise} - Promise for the command.
  */
-export default function init({ parsedArguments, parsedOptions }) {
+export default function init({ parsedArguments, parsedOptions, directory }) {
     const { list, force } = parsedOptions.options;
-    const { name, template, version } = parsedArguments.arguments;
+    const { template, version } = parsedArguments.arguments;
```

Comment:
The `new` command is an alias to `init` that adds name. 
https://github.com/rocjs/roc/blob/master/docs/DefaultCommands.md#new